apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.cs

```diff
@@ -189,6 +191,26 @@ public System.Type DefaultClrType
             }
         }
 
+        /// <summary>
+        /// Converts a <see cref="DbType"/> to the corresponding <see cref="SpannerDbType"/>
+        /// </summary>
+        internal static SpannerDbType FromDbType(DbType dbType)
+        {
+            return dbType switch
+            {
+                DbType.Binary => Bytes,
+                DbType.Boolean => Bool,
+                DbType.Date => Date,
+                DbType.DateTime => Timestamp,
+                DbType.Double => Float64,
+                DbType.Int64 => Int64,
+                DbType.VarNumeric => Numeric,
+                DbType.Object => Unspecified,
+                DbType.String => String,
+                _ => throw new ArgumentOutOfRangeException(nameof(DbType), dbType, null),
+            };
+        }
```

Comment:
```suggestion
        internal static SpannerDbType FromDbType(DbType dbType) => dbType switch
        {
            DbType.Binary => Bytes,
            DbType.Boolean => Bool,
            DbType.Date => Date,
            DbType.DateTime => Timestamp,
            DbType.Double => Float64,
            DbType.Int64 => Int64,
            DbType.VarNumeric => Numeric,
            DbType.Object => Unspecified,
            DbType.String => String,
            _ => throw new ArgumentOutOfRangeException(nameof(DbType), dbType, null),
        };
```

(We could do the same for the DbType property, but I'm not suggesting that change in this PR.)